assignment-2/cmmi.md

```diff
@@ -1,2 +1,136 @@
+| RSKM              | Not at all | Identifying potential risks and how to deal with them is not covered in Scrum.      |
+| VAL               | Partially | A form of validation is done through the Sprint Review in which the prodcut is demonstrated to the stakeholders. Other aspects of validation such as training, manufacturing and maintenance are not covered by Scrum. |
+| VER               | Partially | Verification is done in-sprint in the form of testing activities and during the Sprint Review in which the product is demonstrated to its stakeholders. Scrum does not state what kind of methods, such as peer reviews, test methods, etc. should be applied.            |
+| **Level 4**                              |
+| OPP               | Partially | Organizational process performance is covered in a Sprint Retrospective. This retrospective has however little quantitive properties. It is based soley on subjective input of inidividual team members and does not deal with objective measurements. |
+| QPM               | Not at all | Quantitive management of the team's quality and process perfromance is not covered in Scrum.           |
+| **Level 5**                              |
+| CAR               | Not at all | Scrum does not provide a formal process to prevent defects and problems by means of root cause analysis. |
+| OPM               | Partially | The responsibilty of this process is delegated to the Scrum Master who optimizes team performance by removing impediments thus enabling the team to meet its business objectives. There are however, no quantitive properties prescribed. |
+
+## RUP
+
+| Cmmi Process Area | Covered | Rationale  |
+| ------------------| ------- | ---------  |
+| **Level 2**           |
+| CM                | Fully | RUP's configuration and change management fully covers all of this process area's goals            |
+| MA                | Partially | RUP's project management work flow involves an activity for creating a measurement plan although RUP does not exactly specify how this should be communicated to all relevant parties            |
+| PMC               | Fully | The work flow pertaining to project management covers all aspects of monitoring a project against the plan and taking necessary corrective actions              |
+| PP                | Partially | The project management and environment work flows provide adequate support to the project planning with the exception of determining non-software related project attributes such as labor requirements.                 |
+| PPQA              | Fully | The review record in each of RUP's work flow addresses the issue of quality assurance and the creation of the problem resolution plan insures that all quality issues are dealt with.            |
+| REQM              | Fully | The requirements work flow integrates change management along with the capturing and analyzing of requirements.            |
```

Comment:
Does it integrate change management with capturing and analyzing of requirements? Because then the word along can be left out.